RetrospectiveExtension.Frontend/components/feedbackBoardMetadataForm.tsx

```diff
@@ -105,6 +114,10 @@ class FeedbackBoardMetadataForm extends React.Component<IFeedbackBoardMetadataFo
           this.props.currentBoard.shouldShowFeedbackAfterCollect ?
             this.props.currentBoard.shouldShowFeedbackAfterCollect : false
         ),
+      allowCrossColumnGroups: this.props.isNewBoardCreation ? false :
```

Comment:
hmm it seems like this could evaluate to 

```
allowCrossColumnGroups: !this.props.isNewBoardCreation && this.props.currentBoard.allowCrossColumnGroups
```

Reply:
I'll map this pattern to the others !